src/common/common.reducers.js

```diff
@@ -0,0 +1,11 @@
+import * as constants from './common.actionTypes.js'
+
+export default function (state = {}, action) {
+    switch (action.type) {
+    case constants.APOLOGIZE:
+        console.error(action.message) // eslint-disable-line no-console
```

Comment:
I think its just not implemented. Looks like it's supposed to show some kind of error message to the user. I'll store it in the state for now, it that ok ?